### PR TITLE
Add session-bus to make system-tray icon work

### PR DESCRIPTION
**Pull request by kevinsmia1939**
Add session-bus to make system-tray icon work.

https://github.com/flathub/com.synology.SynologyDrive/pull/8
Not sure if it is also relate. https://github.com/flathub/com.synology.SynologyDrive/issues/9

### CODE REVIEW

File: com.synology.SynologyDrive.yaml [hunk 1]
```diff
@@ -12,6 +12,8 @@ finish-args:
   - --share=ipc
   - --talk-name=org.kde.StatusNotifierWatcher
   - --talk-name=org.freedesktop.Notifications
+  # Need for system tray icon to work
+  - --socket=session-bus
```
Comment (hfiguiere) [inline com.synology.SynologyDrive.yaml:16]: Please use a specific dbus name instead.

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: com.synology.SynologyDrive.yaml (hunk 1 above)
Comment (hfiguiere) [reply com.synology.SynologyDrive.yaml:16]: You are probably looking for `--own-name=org.kde.StatusNotifierItem-7-1`

**Comment by kevinsmia1939** [comment]
bot, build com.synology.SynologyDrive

File: com.synology.SynologyDrive.yaml (hunk 1 above)
Comment (kevinsmia1939) [reply com.synology.SynologyDrive.yaml]: Thanks, I fixed it.

**Comment by hfiguiere** [comment]
I wonder if this fixes #3 too.

**Pull request merged**